ecl/hqlcpp/hqlttcpp.cpp

```diff
@@ -3821,6 +3818,34 @@ void HqlCppTranslator::convertLogicalToActivities(WorkflowItem & curWorkflow)
 }
 
 //------------------------------------------------------------------------
+static HqlTransformerInfo setResultToExtractTransformerInfo("SetResultToExtractTransformer");
+SetResultToExtractTransformer::SetResultToExtractTransformer()
+: NewHqlTransformer(setResultToExtractTransformerInfo)
+{
+}
+
+
+IHqlExpression * SetResultToExtractTransformer::createTransformed(IHqlExpression * expr)
+{
+    OwnedHqlExpr transformed = PARENT::createTransformed(expr);
+    updateOrphanedSelectors(transformed, expr);
```

Comment:
minor: No harm, but this call isn't needed because this transform doesn't replace any table expressions.  There is a partial comment in doUpdateOrphanedSelectors()